server/controllers/interview.js

```diff
@@ -0,0 +1,111 @@
+// connect to the Interview Model
+const Interview = require("../models/interview");
+const User = require("../models/user");
+
+module.exports.createInterview = async (req, res, next) => {
+  const { guest, date, theme, questions, difficulty } = req.body;
```

Comment:
If you're removing guest from the creation then you should also remove it here

Reply:
Guest has been removed as requested.